wsmaster/che-core-api-installer/src/main/java/org/eclipse/che/api/installer/server/impl/LocalInstallerRegistry.java

```diff
@@ -88,12 +99,13 @@ public Installer getInstaller(String installerKey) throws InstallerException {
     }
 
     @Override
-    public List<Installer> getOrderedInstallers(List<String> installers) throws InstallerException {
+    public List<Installer> getOrderedInstallers(List<String> installerKeys) throws InstallerException {
         Map<InstallerFqn, Installer> sorted = new LinkedHashMap<>();
         Set<InstallerFqn> pending = new HashSet<>();
 
-        for (String installer : installers) {
-            doSort(InstallerFqn.parse(installer), sorted, pending);
+        for (String installer : installerKeys) {
+            InstallerFqn installerFqn = InstallerFqn.parse(installer);
+            doSort(stripOffLatestTag(installerFqn), sorted, pending);
```

Comment:
It is not defined by the interface that sorting will be performed by using latest versions instead of specified.
Please specify it or rename argument to `installer id`.